### DISCUSSION TITLE
Training Collapses with train_text_to_image.py

### DISCUSSION THREAD

**Discussion by nighting0le01** [huggingface/diffusers#7845] [Q&A]
If i use a larger (400,000k image text pairs) dataset and using the training script provided. I see weird collapse on all prompts. I haven't even finished 1 epoch and trian with 1e-5 lr . i start with something like this from pretrained SD1.5
![image](https://github.com/huggingface/diffusers/assets/81643693/7c2fa58c-8f16-47ba-bac8-fece13cd6102)
but it transitions to 
![image](https://github.com/huggingface/diffusers/assets/81643693/bb008999-2164-4162-b874-dd9b5a048f17)
after less than 200 steps at batch size 64
(essentially not even seeing the full data!)
Is there any bug in the train_text_to_image.py?
@sayakpaul  have you seen something like this a collapse like this before?
training on a small subset of around 200 images does not lead to collapse but very blurry results

**Comment by bghira** [comment]
it just looks to me like a very high learning rate for a batch size of 64! are you using lr_scale option? have you tried lower value like 1e-6 or 4e-7 ?
